docs/docs/identity-providers/auth0.md

```diff
@@ -35,7 +35,9 @@ While we do our best to keep our documentation up to date, changes to third-part
    | Application Login URI        | [Authenticate Service URL] (e.g. `https://${authenticate_service_url}`)   |
    | Allowed Callback URLs        | Redirect URL (e.g. `https://${authenticate_service_url}/oauth2/callback`).|
 
-   Make sure to click **Save Changes** at the bottom of the page when you're done.
+1. Under **Advanced Settings** → **Oauth**, confirm that **JSON Web Token (JWT) Signature Algorithm** is set to "RS256".
```

Comment:
```suggestion
1. Under **Advanced Settings** → **auth0**, confirm that **JSON Web Token (JWT) Signature Algorithm** is set to "RS256".
```

Reply:
derp.

Reply:
@travisgroth that's a capital o, not a zero.

Reply:
```suggestion
1. Under **Advanced Settings** → **OAuth**, confirm that **JSON Web Token (JWT) Signature Algorithm** is set to "RS256".
```

Reply:
![image](https://user-images.githubusercontent.com/2349184/157732308-0b9c2d0c-824e-4d83-9d08-04d19ff25191.png)

Reply:
🤦 

Would suggest `oauth` or `OAuth` then